wd_core.au3

```diff
@@ -230,9 +231,9 @@ Func _WD_CreateSession($sCapabilities = Default)
 
```

Comment:
shoul we clean up 
```autoit
$_WD_SESSION_DETAILS = ''
```
?

Reply:
please take a look on: [resetinng saved response details](https://github.com/Danp2/au3WebDriver/pull/428/commits/5c90e66737de60dd0e43c50b5a95b4cbf310feed)